kokoro/builds/build.sh

```diff
@@ -10,17 +10,21 @@ set -e
 if [ -z "${CONAN_PROFILE}" ]; then
   CONAN_PROFILE="$(basename $(dirname "${KOKORO_JOB_NAME}"))"
 
-  # That's a temporary work-around. As long as the kokoro jobs don't carry the
-  # conan profile in its name this mapping will do the trick.
+  # Profile mappings. Use that for testing different profiles
+  # without changing the Kokoro config.
   declare -A profile_mapping=( \
-    [gcp_ubuntu]="clang7_relwithdebinfo" \
-    [gcp_ubuntu_ggp]="ggp_relwithdebinfo" \
-    [gcp_ubuntu_test]="clang7_relwithdebinfo" \
-    [gcp_windows]="msvc2017_relwithdebinfo" \
-    [gcp_windows_test]="msvc2017_relwithdebinfo" \
+    [gcc9_relwithdebinfo]="gcc10_release_public" \
```

Comment:
9==10, is this done on purpose?

Reply:
Yes and no. First I remove all the old mappings. We won't need them anymore. Second I add a temporary commit which adds this mapping from gcc9 to gcc10_public. This is just temporary for testing as long as the job config is not yet submitted.

I will remove the temporary commit before merging this PR.

Reply:
I see, lgtm.